ports/libheif/portfile.cmake

```
@@ -20,6 +20,8 @@ vcpkg_cmake_configure(
         -DWITH_DAV1D=OFF
         -DPKG_CONFIG_USE_CMAKE_PREFIX_PATH=ON
         ${FEATURE_OPTIONS}
+    MAYBE_UNUSED_VARIABLES
+        PKG_CONFIG_USE_CMAKE_PREFIX_PATH    # suppress warning
```

```suggestion
        ${FEATURE_OPTIONS}
```
This option was necessary with libheif 1.16 which had `cmake_minimum_required(VERSION 3.0)`.

---

Done